code-review-chat/CodeReviewChat.ts

```diff
@@ -85,8 +87,12 @@ export class CodeReviewChatDeleter extends Chatter {
 			throw Error('Error getting channel history');
 		}
 		const messages = response.messages as SlackMessage[];
+
 		const messagesToDelete = messages.filter((message) => {
 			const isCodeReviewMessage = message.text.includes(this.prUrl);
+			if (message.subtype === 'tombstone') {
```

Comment:
what in the world is `tombstone` 😆

Reply:
That's what slack calls those messages that show when you have a thread on a deleted message